Source/SwiftLintFramework/Configuration+CommandLine.swift

```diff
@@ -39,6 +39,42 @@ private func scriptInputFiles() throws -> [SwiftLintFile] {
     }
 }
 
+private func scriptInputFileLists() throws -> [SwiftLintFile] {
```

Comment:
```suggestion
private func readFilesFromScriptInputFileLists() throws -> [SwiftLintFile] {
```

Please rename the existing method accordingly.